### PR TITLE
[caddx] Correct thing status on bridge initialization

### PR DESCRIPTION
**Pull request by jossuar**
On openhab start/restart the things stayed offline until a message arrived from the alarm panel. Changed the behavior by requesting from the panel the state of the things on bridge initialization. 
Also improved the alarm panel messages in case of unknown events.

### CODE REVIEW

File: bundles/org.openhab.binding.caddx/src/main/java/org/openhab/binding/caddx/internal/handler/LogEventMessage.java [hunk 1]
```diff
@@ -56,33 +56,34 @@ public String toString() {
...
-                    sb.append(" Zone ").append(Integer.parseInt(zud) + 1);
-                    break;
-                case User:
-                    sb.append(" User ").append(Integer.parseInt(zud) + 1);
-                    break;
-                case Device:
-                    sb.append(" Device ").append(zud);
-                    break;
+                switch (logEventType.zud) {
+                    case None:
```
Comment (fwolter) [inline bundles/org.openhab.binding.caddx/src/main/java/org/openhab/binding/caddx/internal/handler/LogEventMessage.java:75]: These should be all upper case. Would be awesome if you could file a follow-up PR!

File: bundles/org.openhab.binding.caddx/src/main/java/org/openhab/binding/caddx/internal/handler/ThingHandlerZone.java [hunk 2]
```diff
@@ -52,7 +52,7 @@ public ThingHandlerZone(Thing thing) {
     @Override
     public void updateChannel(ChannelUID channelUID, String data) {
         if (channelUID.getId().equals(CaddxBindingConstants.ZONE_NAME)) {
-            getThing().setLabel(data);
+            // getThing().setLabel(data);
```
Comment (fwolter) [inline bundles/org.openhab.binding.caddx/src/main/java/org/openhab/binding/caddx/internal/handler/ThingHandlerZone.java:55]: Is this change intentional?

**Review by fwolter — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.caddx/src/main/java/org/openhab/binding/caddx/internal/handler/LogEventMessage.java (hunk 1 above)
Comment (jossuar) [reply bundles/org.openhab.binding.caddx/src/main/java/org/openhab/binding/caddx/internal/handler/LogEventMessage.java:75]: Yes. will do.

File: bundles/org.openhab.binding.caddx/src/main/java/org/openhab/binding/caddx/internal/handler/ThingHandlerZone.java (hunk 2 above)
Comment (jossuar) [reply bundles/org.openhab.binding.caddx/src/main/java/org/openhab/binding/caddx/internal/handler/ThingHandlerZone.java:55]: Yes. The zone name is also exposed as a channel that can be used.

Comment (fwolter) [reply bundles/org.openhab.binding.caddx/src/main/java/org/openhab/binding/caddx/internal/handler/ThingHandlerZone.java:55]: Can the comment be removed, then?

Comment (jossuar) [reply bundles/org.openhab.binding.caddx/src/main/java/org/openhab/binding/caddx/internal/handler/ThingHandlerZone.java]: Done

**Review by fwolter — APPROVED** [review]
